src/components/relationships-panel/package.json

```diff
@@ -8,6 +8,6 @@
     "@bcrs-shared-components/enums": "^2.0.0"
   },
   "devDependencies": {
-    "vue-property-decorator": "^9.1.2"
+    "vue-facing-decorator": "^3.0.3"
```

Comment:
Missed this in my RelationshipsPanel PR (`vue-facing-decorator` is imported in the component)

Reply:
Isn't it one or the other?

Reply:
Yes, we should be using `vue-facing-decorator` for Vue 3. Here I replace `vue-property-decorator` with `vue-facing-decorator`

Reply:
Ok, gtg